### PR TITLE
Roles - Add arsey to fixed-wing

### PR DESCRIPTION
**Pull request by synixebrett**

### CODE REVIEW

File: guides/roles/readme.md [hunk 1]
```diff
@@ -56,7 +56,8 @@
 *Instructors*
 `Brett`  
 *Qualified*
-`Gabriel`
+`Gabe`
```
Comment (nameless) [inline guides/roles/readme.md]:
```suggestion
`Gabe  `
```

Comment (synixebrett) [reply guides/roles/readme.md]: ?

Comment (nameless) [reply guides/roles/readme.md]: spaces

Comment (synixebrett) [reply guides/roles/readme.md]: it doesn't need them

**Pull request merged**